nvtabular/loader/tensorflow.py

```diff
@@ -33,12 +32,21 @@
 # pylint: disable=no-value-for-parameter,unexpected-keyword-arg,redundant-keyword-arg
 
 
-def _validate_dataset(paths_or_dataset, batch_size, buffer_size, engine, reader_kwargs):
+dd_engine = {
+    "parquet": dd.read_parquet,
+    "csv": dd.read_csv,
+    "df": dd.DataFrame,
+}
+
+
+def _validate_dataset(
+    paths_or_dataset, batch_size, buffer_size, engine, reader_kwargs, dataclass=None
```

Comment:
I think this extra parameter came from an intermediate version of the code, but doesn't look like it gets used here:
```suggestion
    paths_or_dataset, batch_size, buffer_size, engine, reader_kwargs
```